web/portal/src/main/webapp/groovy/portal/webui/container/UITabContainer.gtmpl

```diff
@@ -41,8 +41,10 @@
   def modeState = Util.getUIPortalApplication().getModeState();
 %>
 <% if(children != null && children.size() > 0) { %>
-<div class="uiTabContainer" id="$id">
-  <ul class="nav nav-tabs">
+<div id="$id" class="docs-wrapper">
+  <div class="VuetifyApp">
+                      	<div id="tab$id">
+    							      <v-tabs class="docTabs">
```

Comment:
you should remove mention to docs here since it is a geniric container

Reply:
Done